solr/core/src/java/org/apache/solr/core/SolrConfig.java

```diff
@@ -622,15 +622,22 @@ public HttpCachingConfig getHttpCachingConfig() {
 
   public static class HttpCachingConfig implements MapSerializable {
 
+    /**
+     * config xpath prefix for getting HTTP Caching options
+     */
+    private static final String CACHE_PRE
```

Comment:
I can't tell where this is used

Reply:
https://github.com/apache/solr/pull/107/files#diff-473fbcdab103c08461ad1b3c3bb1c6d56f1bcd16d6ce341d80855db2cb20a427R591